docs/sidebars.ts

```diff
@@ -15,37 +15,31 @@ const sidebars: SidebarsConfig = {
 			type: "category",
 			label: "Getting Started",
 			items: [{ type: "autogenerated", dirName: "start" }],
-			link: { type: "doc", id: "start/quick-start" },
```

Comment:
These are the real fix. Copliot find the reason and tested the changes looks fine